port/unix/omrsysinfo.c

```diff
@@ -2456,6 +2482,13 @@ omrsysinfo_get_CPU_utilization(struct OMRPortLibrary *portLibrary, struct J9Sysi
 	cpuTime->numberOfCpus = stats.ncpus; /* get the actual number of CPUs against which the time is reported */
 	cpuTime->cpuTime = (stats.user + stats.sys) * NS_PER_CPU_TICK;
 	status = 0;
+	
+#elif defined(J9ZOS390) /* ZOS */
```

Comment:
The comment here is unnecessary as it is implied by the define. Can we remove?

Reply:
That is not quite correct. Please see https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif to correct the

Reply:
Right! Good catch. I wasn't sure what we do for `#elif` so I thought it was incorrect. The document mentions this:

> - #elif and #else should have comments that echo the previous condition.

Does that mean this line should have `/* defined(AIXPPC) */` as the comment since it's the previous condition?